### PR TITLE
Add P1 Performance burndown

### PR DESCRIPTION
**Pull request by klahnakoski**

### CODE REVIEW

**Comment by klahnakoski** [comment]
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/624

**Comment by klahnakoski** [comment]
@davehunt   Please review for visual layout:

https://deploy-preview-650--firefox-health-dashboard.netlify.com/android

File: src/github/GithubGraph.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        ]),
+        priority: PropTypes.arrayOf(PropTypes.string),
+      }),
+    }),
+  ),
+  startDate: PropTypes.string,
+  title: PropTypes.oneOf([PropTypes.string, PropTypes.node]),
+};
+
+export default withErrorBoundary(GithubGraph);
```
Comment (klahnakoski) [inline src/github/GithubGraph.jsx]: @airimovici  This was copied from `<BugzillaGraph>`, with a minor change.

File: src/github/query.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+// EXAMPLE https://github.com/mozilla-mobile/fenix/labels/eng%3Aperformance
+const GITHUB_ISSUES = new Template('https://api.github.com/repos/{{owner}}/{{repo}}/issues');
+
+const BURNDOWN = {
+  owner: 'mozilla-mobile',
+  repo: 'fenix',
+  labels: 'eng:performance',
+  filter: 'all',
+};
```
Comment (klahnakoski) [inline src/github/query.js]: @airimovici  I am not certain how complicated the queries to Github will get.  I put all the filter parameters in here for now.

File: src/github/query.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+async function getIssues({ timeDomain }) {
+  const { labels, filter } = BURNDOWN;
+  const since = Date.newInstance(timeDomain.min).format(ISO_8601);
+
+  const response = await fetchJson(URL({
+    path: GITHUB_ISSUES.expand(BURNDOWN),
```
Comment (klahnakoski) [inline src/github/query.js]: @airimovici   I dislike the fact that parameters (like `owner`, `repo`) are found in the URL.  I wonder if this template expansion should be included in the URL.

File: src/github/GithubGraph.jsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+  render() {
+    const { title } = this.props;
+
+    return <ChartJsWrapper title={title} {...this.state} />;
+  }
+}
+
+GithubGraph.propTypes = {
+  queries: PropTypes.arrayOf(
```
Comment (airimovici) [inline src/github/GithubGraph.jsx]: This queries prop doesn't seem to be used

Comment (klahnakoski) [reply src/github/GithubGraph.jsx]: good catch!  thanks!

**Comment by davehunt** [comment]
Looks good. Only comment is that we reference "Firefox Preview" in other places but "Fenix" in this graph. I think we should be consistent. Otherwise, r+

**Comment by klahnakoski** [comment]
@davehunt thanks for catching that.  fixed.

**Pull request merged**